utils/sane-backends/Makefile

```diff
@@ -10,7 +10,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=sane-backends
 PKG_VERSION:=1.0.31
-PKG_RELEASE:=1
+PKG_RELEASE:=2
```

Comment:
I don't think this is needed. The patch added a define that was always false without uclibc. In the end, it will get the same binary. Anyway, it is not a blocker, specially for master.